include/fizzy/fizzy.h

```diff
@@ -157,6 +157,15 @@ const FizzyModule* fizzy_parse(const uint8_t* wasm_binary, size_t wasm_binary_si
 /// If passed pointer is NULL, has no effect.
 void fizzy_free_module(const FizzyModule* module);
 
+/// Make a copy of a module.
+///
+/// @param  module          Pointer to module. Cannot be NULL.
+/// @returns                Pointer to a newly allocated module, identical to @a module.
```

Comment:
Need to explain in what case null is returned. Also need to be clear what happens to the input, in case of a failure (I assume there is no case where it would be undefined content).